sonarqube/datadog_checks/sonarqube/data/conf.yaml.example

```diff
@@ -0,0 +1,205 @@
+## All options defined here are available to all instances.
+#
+init_config:
+
+    ## @param is_jmx - boolean - required
+    ## Whether or not this file is a configuration for a JMX integration.
+    #
+    is_jmx: false
```

Comment:
true?

Reply:
Or should we include this option in the instances section

Reply:
Yup 👍